storybookjs/storybook
# Taimoor/added examples

Issue:

## What I did

I simply removed the example of `reactPakistan` and added four mores at the bottom of the list, as `reactPakistanCommons`, `reactPakistanUI`, `reactPakistanIcons` & `reactPakistanLogos`.

## How to test

Simply route to the examples page and you will see the newly added content there.

- Is this testable with Jest or Chromatic screenshots? - no
- Does this need a new example in the kitchen sink apps? - no
- Does this need an update to the documentation? - no

If your answer is yes to any of these, please make sure to include it in your PR.
